webhooks-extension/Gopkg.lock

```diff
@@ -90,6 +90,22 @@
   pruneopts = "UT"
   revision = "6f77996f0c42f7b84e5a2b252227263f93432e9b"
 
+[[projects]]
```

Comment:
please keep dependency updates in separate PRs unless related to the code change, this makes it easier to identify potential issues introduced by the new versions